changelog.txt

```diff
@@ -4,6 +4,17 @@ Changelog
 
 The **signac** package follows `semantic versioning <https://semver.org/>`_.
 
+Version 2
+=========
+
+[2.0.0] -- 2021-xx-xx
+---------------------
+
+Removed
++++++++
+
+ - ``Project.config`` is no longer mutable. Use the command line ``$ signac config`` to modify configuration (#608, #246, #244).
```

Comment:
```suggestion
 - The ``Project.config`` property is no longer mutable. Use the command line ``$ signac config`` to modify configuration (#608, #246, #244).
```
Just to make it clear that we are talking about the Python API here, not the project configuration in general.